plugin-dev/Source/Sentry/Private/Android/SentrySubsystemAndroid.h

```diff
@@ -12,6 +12,7 @@ class SentrySubsystemAndroid : public ISentrySubsystem
 	virtual bool IsEnabled() override;
 	virtual ESentryCrashedLastRun IsCrashedLastRun() override;
 	virtual void AddBreadcrumb(USentryBreadcrumb* breadcrumb) override;
```

Comment:
If we're no longer using the "plain" `AddBreadcrumb`, do we need it?

Reply:
It's still a part of the public API so I think we should leave it even though we're not using it internally anymore.